src/routes/console/account/store.ts

```diff
@@ -6,3 +6,7 @@ export const factors = derived(
     page,
     ($page) => $page.data.factors as Models.MfaFactors & { recoveryCode: boolean }
 );
+export const identities = derived(
+    page,
+    ($page) => $page.data.identities.identities as Models.Identity[]
```

Comment:
seeing this, just as a thought.

we have some places with these weird notations `documents.documents`, `users.users`, etc.

we could think about improving some of those :-) not important for this PR

Reply:
Ya good point. We should change them to something like `$page.data.identityList.identities` to match the type `IdentityList`, ya?

Reply:
yeah, something like that 👍🏻